src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_CollectionExpression.cs

```diff
@@ -138,6 +141,25 @@ static BoundNode unwrapListElement(BoundCollectionExpression node, BoundNode ele
             }
         }
 
+        private bool IsSingleReadOnlySpanSpread(BoundCollectionExpression node, [NotNullWhen(true)] out BoundExpression? spreadExpression)
+        {
+            spreadExpression = null;
+
+            if (node is
+                {
+                    Elements:
```

Comment:
nit: can we just access Elements on the LHS of the 'is' to reduce depth of nesting here?